test/integration/test/recipe_test.js

```diff
@@ -0,0 +1,177 @@
+        return chakram.get(Prepper.foxboxManager.serviceListURL);
+      })
+      .then(function(listResponse) {
+        expect(listResponse).to.have.status(200);
+        var isFound = listResponse.body.some(
+          entry => entry.id === 'thinkerbell/First Recipe');
+        expect(isFound).to.be.true;
+      });
+    }); 
+
+    it('Add two recipes one by one',function(){        
+      
+      return chakram.put(Prepper.foxboxManager.setterURL, 
+        generateThinkerbellNewRecipePayload('Second Recipe'))
+        .then(function(cmdResponse) {
+
+          expect(cmdResponse).to.have.status(200);
+          expect(cmdResponse.body['thinkerbell-add-rule']).equals(null);
+          return chakram.put(Prepper.foxboxManager.setterURL, 
+            generateThinkerbellNewRecipePayload('Third Recipe'));
+        })
+        .then(function(cmdResponse) {
+          expect(cmdResponse).to.have.status(200);
+          expect(cmdResponse.body['thinkerbell-add-rule']).equals(null);
+          return chakram.get(Prepper.foxboxManager.serviceListURL);
+        })
+        .then(function(listResponse) {
+          expect(listResponse).to.have.status(200);
+          var isFound = listResponse.body.some(
+            entry => entry.id === 'thinkerbell/Second Recipe');
+          expect(isFound).to.be.true; 
+          isFound = listResponse.body.some(
+            entry => entry.id === 'thinkerbell/Third Recipe');
+          expect(isFound).to.be.true; 
+        });
+      });
+  
+    after(function(){
+      var payload = {'select':
+      {'id': 'thinkerbell/First Recipe/remove'},'value':null};
+      return chakram.put(Prepper.foxboxManager.setterURL, payload)
+      .then(function(cmdResponse) {
+        payload = {'select':
+        {'id': 'thinkerbell/Second Recipe/remove'},'value':null};
+        return chakram.put(Prepper.foxboxManager.setterURL, payload);
+      })
+      .then(function(cmdResponse) {
+        payload = {'select':
+        {'id': 'thinkerbell/Second Recipe/remove'},'value':null};
+        return chakram.put(Prepper.foxboxManager.setterURL, payload);
+      });
+    });
+  });
+
+  describe('remove recipes', function(){
+
+    before(function(){
+      return chakram.put(Prepper.foxboxManager.setterURL, 
+        generateThinkerbellNewRecipePayload('First Recipe'))
+        .then(function(cmdResponse) {
+          expect(cmdResponse).to.have.status(200);
+          return chakram.put(Prepper.foxboxManager.setterURL, 
+            generateThinkerbellNewRecipePayload('Second Recipe'));
+        })
+        .then(function(cmdResponse) {
+          expect(cmdResponse).to.have.status(200);
+          return chakram.put(Prepper.foxboxManager.setterURL, 
+            generateThinkerbellNewRecipePayload('Third Recipe'));
+        });
+    });
+
+    it('Remove single recipe',function(){
+      var payload = {'select':
+      {'id': 'thinkerbell/First Recipe/remove'},'value':null};
```

Comment:
As we define the deletion payload many times, I would define a function `generateDeleteRecipePayload()`